h2o-docs/src/product/welcome.rst

```diff
@@ -526,8 +526,9 @@ Hadoop Launch Parameters
 -  ``-nthreads <maximum number of CPUs>``: Specify the number of CPUs to use. This defaults to using all CPUs on the host, or you can enter a positive integer.
 -  ``-baseport <initialization port for H2O nodes>``: Specify the initialization port for the H2O nodes. The default is ``54321``.
 -  ``-ea``: Enable assertions to verify boolean expressions for error detection.
--  ``-verbose:gc``: Include heap and garbage collection information in the logs.
--  ``-XX:+PrintGCDetails``: Include a short message after each garbage collection.
+-  ``-verbose:gc``: Include heap and garbage collection information in the logs. Deprecated in Java 9, removed in Java 10.
```

Comment:
Just minor detail: would it be better to separate deployment options from jvm-specific options? Seeing baseport followed by `-ea` is surprising ;-)

Reply:
Addressed this in the latest commit.